asm/main_rodata_020908D4.s

```diff
@@ -2933,19 +2933,19 @@ _02098070:
 	.byte 0x45, 0x2F, 0x73, 0x74, 0x5F, 0x69, 0x32, 0x6E
 	.byte 0x5F, 0x6A, 0x2E, 0x62, 0x69, 0x6E, 0x00, 0x00
 #endif
-	.global _0209808C
-_0209808C:
+	.global ITEM_NAME_FORMAT_CSU
+ITEM_NAME_FORMAT_CSU: // "[CS:U]%s[CR]"
 	.byte 0x5B, 0x43, 0x53, 0x3A, 0x55, 0x5D, 0x25, 0x73
 	.byte 0x5B, 0x43, 0x52, 0x5D, 0x00, 0x00, 0x00, 0x00
-	.global _0209809C
-_0209809C:
+	.global ITEM_NAME_FORMAT_CSZ
+ITEM_NAME_FORMAT_CSZ: // "[CS:Z]%s[CR]"
 	.byte 0x5B, 0x43, 0x53, 0x3A, 0x5A, 0x5D, 0x25, 0x73
 	.byte 0x5B, 0x43, 0x52, 0x5D, 0x00, 0x00, 0x00, 0x00
-	.global _020980AC
-_020980AC:
+	.global ITEM_NAME_FORMAT_PLAIN
+ITEM_NAME_FORMAT_PLAIN: // "%s"
 	.byte 0x25, 0x73, 0x00, 0x00
-	.global _020980B0
-_020980B0:
+	.global ITEM_NAME_FORMAT_CSI
+ITEM_NAME_FORMAT_CSI: // "[CS:I]%s[CR]"
```

Comment:
The `CS:` changes the text color. Here the colors used in these strings, courtesy of Morei from the SkyTemple Discord:
```
[CS:I]: cream
[CS:U]: yellow
[CS:Z]: indigo
```
Let's name these symbols after the colors. For example, `ITEM_NAME_FORMAT_CSU` can be `ITEM_NAME_FORMAT_YELLOW`.